### DISCUSSION TITLE
Trouble getting ITN using Pretrained Neural Text Normalization Model

### DISCUSSION THREAD

**Discussion by dylansoemitro** [NVIDIA-NeMo/NeMo#4534] [Q&A] [answered]
Hello, I am trying to use NeMo's neural text normalization models (duplex) for ITN, outlined [here](https://github.com/NVIDIA/NeMo/blob/main/docs/source/nlp/text_normalization/nn_text_normalization.rst#id7). I'm using the [Neural Text Normalization T5](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/nemo/models/neural_text_normalization_t5/version) pre-trained model, but when running inference using mode=itn or mode=joint it still performs TN instead of ITN. The command I am running is this: `python duplex_text_normalization/duplex_text_normalization_infer.py lang=en mode=joint tagger_pretrained_model=neural_text_normalization_t5 decoder_pretrained_model=neural_text_normalization_t5 inference.interactive=True
` I have tried both interactive and inference from a file, and both do not work (performs TN instead). Any pointers would be helpful.

**Comment by ekmb** [comment, marked as answer]
This checkpoint is TN only. For neural ITN we have a pre-trained single-pass [Thutmose Tagger checkpoint](https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/main/nlp/text_normalization/text_normalization_as_tagging.html ), it achieves similar performance as the Duplex architecture on ITN on the Google dataset.

**Reply by dylansoemitro** [reply]
Got it, just wanted to compare the duplex with the Thutmose Tagger. Thank you!
